### DISCUSSION TITLE
Some example projects not playing (3.32)

### DISCUSSION THREAD

**Discussion by prbrowne** [PhaserEditor2D/PhaserEditor2D-v3#188] [General]
Running the example project 'Volcano' in the IDE,  the browser shows only a black screen.  Also happens with the internal browser via F10.

Any help?

**Comment by arianfornaris** [comment]
Hi!

Please, look the Volcano demo is coded in TypeScript. You should build the game first:

```
npm install
npm run build
```

Check the README file of the project.
